### ISSUE TITLE
NTR: Coccidioidomycosis on tissue culture (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8108]
New term request Coccidioidomycosis on tissue culture (nadea3_190719124530): Skin or organ, immitis, posadasii

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000965
- label: Positive tissue coccidioidomycosis culture
- parents: [Positive infectious agent test (HP:0430030)](https://hpo.jax.org/app/browse/term/HP:0430030)
- pmid: 
- def: Growth of coccidioidomycosis in a culture derived from skin or an organ.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8108
- synonyms:
